### DISCUSSION TITLE
Dashboard Grid/Metrics: Allow selecting portion of graph

### DISCUSSION THREAD

**Discussion by natehitze-eventlink** [uptrace/uptrace#328] [Ideas]
Sometimes I'm looking at a wide range of time to notice a spike in metrics and then need to narrow in around the time of the spike. It would be nice to be able to select a portion of the graph to set the start/end times for the filter. Right now I have to hover my mouse over the graph to the find the times and then type them into the "From date" and "To date" filter.

**Comment by vmihailenco** [comment]
I've created an [issue](https://github.com/uptrace/uptrace/issues/329) to track this. It is a proper medium sized feature so hopefully we will have something in February.
